src/task2/controllers/userController.js

```diff
@@ -0,0 +1,16 @@
+import {getUsers, getUserById, addUser, updateUser, deleteUser } from '../db/users';
+import { PAGE_LIMIT } from '../config/global';
+
+export const get = (req, res) => {
```

Comment:
Consider exporting all the needed things as a part of the object at the bottom of the file - thus creating a single export place like an API for your module.